### PR TITLE
Feat/ forget password UI

### PR DESCRIPTION
**Pull request by giorgibarishvili**
# Pull Request Template for ArtOa

## Description

Please provide a summary of the changes in this pull request:
https://tteam1.atlassian.net/jira/software/projects/SCRUM/boards/1?selectedIssue=SCRUM-94
completed UI of Forget password part.

- What does this PR do?
- Why are these changes necessary?
- Any relevant links or references?

Fixes: [issue number/link]

---

## Type of Change

Please select the type of change this PR introduces (tick the appropriate box):

- [ ] 🐛 Bug fix (non-breaking change that fixes an issue)
- [x] ✨ New feature (non-breaking change that adds functionality)
- [ ] 🚨 Breaking change (fix or feature that would cause existing functionality
      to not work as expected)
- [ ] 🔄 Refactor (code or file structure changes without affecting functionality)
- [ ] 📝 Documentation update (changes or additions to documentation)
- [ ] 🚀 Performance improvement (optimization in code or resources)

---

## Checklist

Please ensure the following tasks have been completed before requesting a review:

- [x] My code follows the style guidelines of this project
- [ ] I have run tests to ensure my changes work as expected
- [ ] I have added/updated necessary tests or documentation (if applicable)
- [x] I have self-reviewed my code and refactored any areas that need improvement
- [x] There are no console errors or warnings related to my changes
- [x] The PR includes a clear, detailed title and description
- [x] Any relevant issues have been linked or mentioned in this PR
- [x] I have reviewed the acceptance criteria and met all requirements
- [x] I have considered edge cases and errors that may occur due to my changes

---

## Testing Instructions

Please describe how your changes can be tested:

1. **Test Setup:** (Any steps for environment setup)
2. **Test Steps:**
   - Step 1: ...
   - Step 2: ...
   - ...

---

## Screenshots / Visual Changes

Please attach any relevant screenshots, videos, or GIFs to illustrate the impact
of this change (if applicable).

---

## Additional Notes

Is there anything else the reviewer should know about this PR? Any known issues,
challenges, or decisions made during implementation?

### CODE REVIEW

File: src/app/(auth)/forget-pass/page.tsx
Comment (mshubitidze) [inline src/app/(auth)/forget-pass/page.tsx:1]: "/forgot-password" might be a better name for this route

File: src/app/(auth)/sign-in/_components/sign-in-form.client.tsx [hunk 1]
```diff
@@ -16,17 +16,21 @@ export function SignInForm() {
...
-        </Link>
+        <div className="mt-1 flex justify-between">
+          <div className="flex items-center">
+            <Checkbox className="me-2 size-5" /> Remember Me
+          </div>
+          <Link
+            className="flex items-center text-xs text-muted-foreground"
+            href="/forget-pass"
+          >
+            Forget Password
```
Comment (mshubitidze) [inline src/app/(auth)/sign-in/_components/sign-in-form.client.tsx:27]: `Forgot Password`, instead of `Forget`

**Review by mshubitidze — CHANGES_REQUESTED** [review]
lgtm 🚀
please merge main into this branch

**Review by mshubitidze — APPROVED** [review]
